_pages/en/strategy-build-vs-buy.md

```diff
@@ -67,48 +71,60 @@ All ESDC personnel involved in the exploration of IT Solutions and in IT investm
-  - [Appendix A: Mandatory Procedures for Enterprise Architecture Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602)
-  - [Appendix B: Mandatory Procedures on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604)
-- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
-- GoC [Architecture Principles Standards](https://wiki.gccollab.ca/Government_of_Canada_Architectural_Standards)
+- Government of Canada (GoC)
+  - [Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html)
+- Treasury Board of Canada Secretariat (TBS)
+  - TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601); and specifically:
+    - [Appendix A: Mandatory Procedures for Enterprise Architecture Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602)
+    - [Appendix B: Mandatory Procedures on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604)
+  - TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+  - GoC [Architecture Principles Standards](https://wiki.gccollab.ca/Government_of_Canada_Architectural_Standards)
+- Employment and Social Development Canada (ESDC)
+  - ESDC [Policy on Project and Programme Management (PPPM)](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Policy%20on%20Project%20and%20Programme%20Management.pdf)
+  - ESDC Information Management Policy (being drafted)
+  - ESDC [Procurement Policies](http://iservice.prv/eng/finance/purchasing/policy.shtml)
+  - ESDC Security Policy (being drafted)
+  - ESDC [Target Solution Delivery Model](strategy-target-solution-delivery-model.html) Strategy (draft in consultation)
+  - ESDC Data Strategy
 
 ### Governance, Compliance, and Reporting
 
-1. IT Solutions are broken down into manageable parts (IT Products) in line with the [GoC Business Capability Model](https://gcconnex.gc.ca/file/group/21723432/al#3l3319941)
-2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (to be created - see [Coherent set of actions](#coherent-set-of-actions) and read [Managing the Risks of Obtaining of Software](#managing-the-risks-of-obtaining-of-software)).
-3. IT Product / IT Solution maintenance and support may be done in-house or outsourced to a 3rd party vendor
-   - Outsourced maintenance and support may be given to a 3rd party different than the one providing the software licence.
+1. IT Solutions are broken down into manageable parts (IT Products) in line with the the ESDC Business Capability Model, whichever is most relevant.
+2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (to be created - see [Coherent set of actions](#coherent-set-of-actions) and read [Managing the Risks of Acquiring of Software](#managing-the-risks-of-acquiring-of-software)).
```

Comment:
link to "Managing the risks of acquiring of software" is broken (tried it in a Jekyll rendered site).

Reply:
Will fix